config/routes.rb

```diff
@@ -1,4 +1,23 @@
 Rails.application.routes.draw do
+  devise_for :users
+  root "packages#index"
+  # Routes for the Package resource:
+
+  # CREATE
+  post("/insert_package", { :controller => "packages", :action => "create" })
+          
+  # READ
+  get("/packages", { :controller => "packages", :action => "index" })
+  
+  # UPDATE
+  
+  post("/modify_package/:path_id", { :controller => "packages", :action => "update" })
```

Comment:
I like how you are descriptive with your routes but if this was used as an api it would be difficult to track the endpoints so keeping them restful will be your best friend. 

TLDR: You can have multiple routes with the same name as long as the http verbs are different: 

get('/packages', ...)
post('/packages', ...)

Reply:
Yes! Learned about that right after I submitted